src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -1104,23 +1103,11 @@ public ImmutableArray<SymbolDisplayPart> ToDisplayParts(SymbolDisplayFormat form
             CSharpSyntaxNode block,
             CSharpSyntaxNode expression,
             CSharpSyntaxNode syntax,
-            DiagnosticBag diagnostics)
+            DiagnosticBag diagnostics,
+            ErrorCode code)
```

Comment:
I agree we use default values.  However, we also get bugs because of it :)

The default value does nothing (IMO) here but open up the possibility of someone passing in the wrong value.  By being required, that possibility is avoided entirely.  To me, that seems valuable.  Why even introduce that possibility?

I mean, i'm leaning at this point to just removing both errorcodes and just having a generic: "Block bodies and expression bodies cannot both be provided."  I don't think there's much value in having specific error codes for these different symbols kinds.